### PR TITLE
Allow creating/updating permits with description

### PR DESCRIPTION
**Pull request by mingfeng**
Refs: PV-372

### CODE REVIEW

File: src/components/residentPermit/EditResidentPermitPreview.tsx [hunk 1]
```diff
@@ -59,7 +59,9 @@ const EditResidentPermitPreview = ({
           {t(`${T_PATH}.goBack`)}
         </Button>
         <Button
-          disabled={!(refundAccountNumber && isValidIBAN(refundAccountNumber))}
```
Comment (amanpdyadav) [inline src/components/residentPermit/EditResidentPermitPreview.tsx:62]: Should support only Finnish IBAN

**Review by amanpdyadav — CHANGES_REQUESTED** [review]

File: src/components/residentPermit/EditResidentPermitPreview.tsx (hunk 1 above)
Comment (mingfeng) [reply src/components/residentPermit/EditResidentPermitPreview.tsx:62]: I have updated the PR regarding this, but this line is not changed. So, what I did is creating our own version of `isValidIBAN` that checks if the IBAN number starts with "FI" and then calls the `isValidIBAN` function from` ibantools`.

File: src/utils.ts [hunk 2]
```diff
@@ -218,3 +219,7 @@ export function getPermitTotalPrice(
     0
   );
 }
+
+export function isValidIBAN(value: string): boolean {
+  return value.startsWith('FI') && _isValidIBAN(value);
```
Comment (amanpdyadav) [inline src/utils.ts:224]: Why not to use the library inbuilt methods for checking it?
https://github.com/City-of-Helsinki/parking-permits-ui/commit/32c99d2e281404206e66f36b300a62ca2ba89398

**Review by amanpdyadav — APPROVED** [review]
LGTM 👍

**Pull request merged**